Rogue-Ape-Studios/Crusader
# ✨ Item System List

## Content

Implements easily refreshing the list of items.

## Changes

### Added
`Assets/Prefabs/Items/AttackSpeedUp.asset` : Was created, for improving attack speed.
`Assets/Prefabs/Items/Attack Damage Up.asset` : Was created for improving attack damage.
`Assets/Scripts/Items/Manager/Editor/ItemManagerEditor.cs` : Was created for improved developer experience.
`Assets/Prefabs/Items/Attack Damage Up.asset` : Was created for improving attack damage.
`Assets/Scripts/Items/Manager/Editor/RogueApeStudio.Crusader.Items.Manager.Editor.asmdef` : Was created.

### Updated
`Assets/Prefabs/Items/ItemManager.prefab` : Was updated to support the new items.
`Assets/Scripts/Items/Manager/ItemManager.cs` : Was updated with a public method to support updating the list.

## Testing

The feature / Bugfix can be testing by following these steps:

1. Remove an item from the list in `ItemManager`.
2. Press the refresh list button.
3. Observe the list being refilled.
